docs/services/motion/_index.md

```diff
@@ -0,0 +1,466 @@
+Note that `arm.MoveToPosition` does not use `world_state`, so collision checking and obstacle avoidance *will not* be performed.
+
+If you need collision checking and obstacle avoidance, use [`Move`](#move).
+
+{{% /alert %}}
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `component_name` ([ResourceName](https://python.viam.dev/autoapi/viam/gen/common/v1/common_pb2/index.html#viam.gen.common.v1.common_pb2.ResourceName)): The name of the component to move.
+  This component must support the `move_to_position` API call with a Pose.
+  As of April 21, 2023, [arm](/components/arm/) is the only component so supported.
+
+- `destination` ([PoseInFrame](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.PoseInFrame)):
+  Describes where the `component_name` should end up.
+  Can be any pose, from the perspective of any component whose location is configured as a [`frame`](../frame-system/).
+  The `destination` will be converted into the frame of the component specified in `component_name` when passed to `move_to_position`.
+
+- `world_state` ([WorldState](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.WorldState)) (*optional*): Data structure specifying information about the world around the robot.
+  **See [note above](#move-vs-movetoposition); a custom implementation of `MoveToPosition` may or may not make use of `world_state`.**
+
+  `world_state` includes obstacles and transforms:
+  - **Obstacles**: Geometries located at a pose relative to some frame.
+    When solving a motion plan with movable frames that contain inherent geometries, the solved path is constrained such that none of those inherent geometries intersect with the obstacles.
+    Important considerations:
+    - If a motion begins with a component already in collision with an obstacle, collisions between that specific component and that obstacle will not be checked.
+    - The Motion Service assumes that obstacles with mobile parents move along with their parents while solving.
+      This ensures that obstacles that are temporarily attached to moving components do not cause collisions during movement.
+    - Geometries are "part of" their frame, rather than at the distal end of the frame.
+      Their poses are relative to the *origin* of the specified frame.
+      A geometry associated with the frame of an arm with a pose of {X: 0, Y: 0, Z: -10} will be interpreted as being 10mm below the base of the arm, not 10mm below the end effector.
+      This is different from `destination` and `component_name`, where poses are relative to the distal end of a frame.
+  - **Transforms**: These are a list of `PoseInFrame` messages that specify arbitrary other transformations that will be ephemerally added to the frame system at solve time.
```

Comment:
Note to self: change this in the same way that line 147 is changed.

Reply:
```suggestion
  - **Transforms**: A list of `PoseInFrame` messages that specify other transformations to temporarily add to the frame system at solve time.
  Transforms can be used to account for geometries that are attached to the robot but not configured as robot components.
  For example, you could use a transform to represent the volume of a marker held in your robot's gripper.
  Transforms are not added to the config or carried into later processes.
```

Reply:
NVM; after talking to Peter I will just remove the details from this and say "not used"